client/src/components/SignUp.tsx

```
@@ -183,15 +207,29 @@ const SignUp = () => {
             className="form-input p-2 w-full"
           />
+          <span
+            className="absolute right-3 top-3 cursor-pointer text-gray-600"
+            onClick={toggleConfirmPasswordVisibility}
+            onKeyDown={(event) => {
+              if (event.key === "Enter" || event.key === " ") {
+                toggleConfirmPasswordVisibility();
+              }
+            }}
+            tabIndex={0}
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)